src/view/Template.php

```diff
@@ -25,10 +22,8 @@
  *
  * @property string|bool $viewFile The view file currently being rendered. False if no view file is being
  * rendered. This property is read-only.
- *
- * @author Qiang Xue <qiang.xue@gmail.com>
  */
-class View extends Component implements DynamicContentAwareInterface
+class Template implements DynamicContentAwareInterface
```

Comment:
This class name is confusing to me (and was before).
According to its function it is TemplateRenderer and not a single Template.